### PR TITLE
Issue with NPE in person example fixed.  Also updated to change in ja…

### PR DESCRIPTION
**Pull request by grechaw**
Java Client API had a breaking change last night which changes the number of args in some lambda expressions.  This PR fixes that (in PersonExplorer).

Also, there was an NPE issue #201  that is also fixed here.

### CODE REVIEW

File: entity-services-examples/src/main/java/com/marklogic/entityservices/examples/person/PersonExplorer.java [hunk 1]
```diff
@@ -187,7 +193,7 @@ public void runOpticQuery() {
 //                .select(pb.col("firstName"));
 
         Plan p = pb.fromTriples(pb.pattern(pb.subjects(pb.col("s")),
-            pb.predicates(pb.sem.iri("http://www.w3.org/1999/02/22-rdf-syntax-ns#")),
+            pb.predicates(pb.sem.iri("http://www.w3.org/1999/02/22-rdf-syntax-ns#type")),
```
Comment (bsrikan) [inline entity-services-examples/src/main/java/com/marklogic/entityservices/examples/person/PersonExplorer.java]: What does this line do @grechaw by adding "type"

File: entity-services-examples/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Mon Oct 24 10:09:58 PDT 2016
+#Tue Nov 08 15:53:05 PST 2016
```
Comment (bsrikan) [inline entity-services-examples/gradle/wrapper/gradle-wrapper.properties]: Is there any change required to e2e gradle-wrapper.properties @grechaw ?

Comment (grechaw) [inline entity-services-examples/gradle/wrapper/gradle-wrapper.properties]: no, this kind of change just happens when you run gradle wrapper.  I could have excluded it from this PR, it's a no-op.

File: entity-services-examples/src/main/java/com/marklogic/entityservices/examples/person/PersonExplorer.java (hunk 1 above)
Comment (grechaw) [inline entity-services-examples/src/main/java/com/marklogic/entityservices/examples/person/PersonExplorer.java]: Good question.  without the 'type' here it's a bug.  Adding 'type' turns this IRI from garbage into rdf:type, which is what we wanted to query for.

**Pull request merged**